ppr-ui/src/composables/pprRegistration/usePprRegistration.ts

```diff
@@ -133,7 +135,13 @@ export const usePprRegistration = () => {
     }
   }
 
+  /** Returns true when Security Act Notice Feature Flag is enabled **/
+  const isSecurityActNoticeEnabled: ComputedRef<boolean> = computed((): boolean => {
+    return getFeatureFlag('ppr-sa-notice-enabled')
+  })
+
```

Comment:
@doug-lovett Will include the user profile flag/indicator here in this condition when available